lupa/_lupa.pyx

```diff
@@ -1399,12 +1402,13 @@ cdef bint call_python(LuaRuntime runtime, lua_State *L, py_object* py_obj) excep
 
 cdef int py_call_with_gil(lua_State* L, py_object *py_obj) with gil:
     cdef LuaRuntime runtime = None
+    runtime = <LuaRuntime?>py_obj.runtime
     try:
-        runtime = <LuaRuntime?>py_obj.runtime
         return call_python(runtime, L, py_obj)
     except:
-        try: runtime.store_raised_exception()
```

Comment:
We have to put something on stack for lua to raise instead of python's exception then, and I'm not sure what? "error during Python call" again?